scripts/modules/b2b-account/quotes.js

```diff
@@ -206,6 +206,9 @@ define([
         filter: function(collection) {
             var filterStr = "";
             var qName = $("#searchName").val();
+            if (typeof String.prototype.mzCheckSpacialChar === 'function') {
```

Comment:
since mzCheckSpacialChar is part of SDK then it will always be available. so no need to check this is available or not but not 100% sure about it.

Reply:
Yes, I added this for safer side if function is not loaded by sdk current  search functionality should work, but in other side if SDK will not load mozu application will not work so I am fine to remove this if condition.